### DISCUSSION TITLE
How to disable visualstudiocode's plugin "C/C++" "IntelliSenseMode" edit a specific file

### DISCUSSION THREAD

**Discussion by Huaxidesu** [microsoft/vscode-cpptools#9919] [Q&A]
Visualstudiocode will automatically modify my "c_cpp_properties.json"(I am using clang and it works fine), what can I do to prevent him from modifying this file instead of turning off the "IntelliSenseMode" function and del path "MSVC".(I set that file to "Reading Mode", but every time I open a large number of VSC prompts make me troublesome)
`[2022/9/15 23:37:04] 对于 C++ 源文件，IntelliSenseMode 已根据编译器参数和查询 compilerPath 从“windows-clang-x64”更改为“windows-msvc-x64”:“D:/Office/Library/Microsoft/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.33.31629/bin/Hostx64/x64/cl.exe” [2022/9/15 23:37:04] 对于 C 源文件，IntelliSenseMode 已根据编译器参数和查询 compilerPath 从“windows-clang-x64”更改为“windows-msvc-x64”:“D:/Office/Library/Microsoft/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.33.31629/bin/Hostx64/x64/cl.exe”`

**Comment by sealpp** [comment]
same problems here, did u solved it?
